WcaOnRails/app/webpacker/lib/requests/routes.js.erb

```diff
@@ -34,6 +34,8 @@ export const competitionReportUrl = (id) => `<%= CGI.unescape(Rails.application.
 
 export const competitionApiUrl = (id) => `<%= CGI.unescape(Rails.application.routes.url_helpers.api_v0_competition_path("${id}"))%>`;
 
+export const competitionsApiUrl = `<%= CGI.unescape(Rails.application.routes.url_helpers.api_v0_competitions_path)%>`;
```

Comment:
Maybe you can initialize this inside `apiV0Urls` which is there at bottom of this file.

Reply:
Oh my, my bad, my local branch has not fetched from main for so long that that part is entirely not there. I see what you mean. Yeah it would make sense to put that there. But why aren't all the api urls in that object? Is that something to be done?

Reply:
Actually I created that object recently because it is easy to understand the purpose from the path. Changing all existing APIs under this is really difficult. So, maybe we can add the new ones here and eventually change the old ones when we change something in them. Also, in long term, I don't know whether we even have this `routes.js.erb`, maybe we will find some other routing way.

Reply:
done. not sure if calling these URLs 'list' is appropriate